server/service/osquery_test.go

```diff
@@ -1634,7 +1634,7 @@ func TestDetailQueriesWithEmptyStrings(t *testing.T) {
       "cpu_subtype": "Intel x86-64h Haswell",
       "cpu_type": "x86_64h",
       "hardware_model": "MacBookPro11,4",
-      "hardware_serial": "NEW_HW_SERIAL",
+      "hardware_serial": "NEW_HW_SRL",
```

Comment:
because you're now filtering out any serial # that includes the string "serial"?

Reply:
Correc